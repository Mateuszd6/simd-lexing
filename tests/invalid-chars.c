int main(int i1 = 0, int i2 = 0, int i3 = 0,
         int i4 = 0, int i5 = 0, int i6 = 0,
         int i7 = 0, int i8 = 0, int i9 = 0)
{
    char* foo = "q";
    int xx = 1;
    int32_t q = 0;

    char c1 = '1';
    char c2 = 'a';
    char c3 = '0';
    char c4 = '\0';
    char c5 = '\n';
    char c6 = '\r';
    char c7 = '\000';
    char c8 = '\ThisIsTooLong';

    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;
    long; long; long; long; long; long; long; long; long; long;

    return q;
}
