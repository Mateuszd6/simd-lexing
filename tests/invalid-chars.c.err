../tests/invalid-chars.c:16:15: error: Invalid character literal
