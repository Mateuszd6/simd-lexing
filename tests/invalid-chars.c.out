../tests/invalid-chars.c:1:1: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:1:5: TOK 0 (L = 4): "main"
../tests/invalid-chars.c:1:9: TOK 1 (L = 1): "("
../tests/invalid-chars.c:1:10: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:1:14: TOK 0 (L = 2): "i1"
../tests/invalid-chars.c:1:17: TOK 1 (L = 1): "="
../tests/invalid-chars.c:1:19: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:1:20: TOK 1 (L = 1): ","
../tests/invalid-chars.c:1:22: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:1:26: TOK 0 (L = 2): "i2"
../tests/invalid-chars.c:1:29: TOK 1 (L = 1): "="
../tests/invalid-chars.c:1:31: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:1:32: TOK 1 (L = 1): ","
../tests/invalid-chars.c:1:34: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:1:38: TOK 0 (L = 2): "i3"
../tests/invalid-chars.c:1:41: TOK 1 (L = 1): "="
../tests/invalid-chars.c:1:43: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:1:44: TOK 1 (L = 1): ","
../tests/invalid-chars.c:2:10: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:2:14: TOK 0 (L = 2): "i4"
../tests/invalid-chars.c:2:17: TOK 1 (L = 1): "="
../tests/invalid-chars.c:2:19: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:2:20: TOK 1 (L = 1): ","
../tests/invalid-chars.c:2:22: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:2:26: TOK 0 (L = 2): "i5"
../tests/invalid-chars.c:2:29: TOK 1 (L = 1): "="
../tests/invalid-chars.c:2:31: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:2:32: TOK 1 (L = 1): ","
../tests/invalid-chars.c:2:34: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:2:38: TOK 0 (L = 2): "i6"
../tests/invalid-chars.c:2:41: TOK 1 (L = 1): "="
../tests/invalid-chars.c:2:43: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:2:44: TOK 1 (L = 1): ","
../tests/invalid-chars.c:3:10: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:3:14: TOK 0 (L = 2): "i7"
../tests/invalid-chars.c:3:17: TOK 1 (L = 1): "="
../tests/invalid-chars.c:3:19: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:3:20: TOK 1 (L = 1): ","
../tests/invalid-chars.c:3:22: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:3:26: TOK 0 (L = 2): "i8"
../tests/invalid-chars.c:3:29: TOK 1 (L = 1): "="
../tests/invalid-chars.c:3:31: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:3:32: TOK 1 (L = 1): ","
../tests/invalid-chars.c:3:34: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:3:38: TOK 0 (L = 2): "i9"
../tests/invalid-chars.c:3:41: TOK 1 (L = 1): "="
../tests/invalid-chars.c:3:43: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:3:44: TOK 1 (L = 1): ")"
../tests/invalid-chars.c:4:1: TOK 1 (L = 1): "{"
../tests/invalid-chars.c:5:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:5:9: TOK 1 (L = 1): "*"
../tests/invalid-chars.c:5:11: TOK 0 (L = 3): "foo"
../tests/invalid-chars.c:5:15: TOK 1 (L = 1): "="
../tests/invalid-chars.c:5:18: TOK 5 (L = 1): "q"
../tests/invalid-chars.c:5:20: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:6:5: TOK 0 (L = 3): "int"
../tests/invalid-chars.c:6:9: TOK 0 (L = 2): "xx"
../tests/invalid-chars.c:6:12: TOK 1 (L = 1): "="
../tests/invalid-chars.c:6:14: TOK 2 (L = 1): "1"
../tests/invalid-chars.c:6:15: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:7:5: TOK 0 (L = 7): "int32_t"
../tests/invalid-chars.c:7:13: TOK 0 (L = 1): "q"
../tests/invalid-chars.c:7:15: TOK 1 (L = 1): "="
../tests/invalid-chars.c:7:17: TOK 2 (L = 1): "0"
../tests/invalid-chars.c:7:18: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:9:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:9:10: TOK 0 (L = 2): "c1"
../tests/invalid-chars.c:9:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:9:15: TOK 4 (L = 1): "1"
../tests/invalid-chars.c:9:18: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:10:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:10:10: TOK 0 (L = 2): "c2"
../tests/invalid-chars.c:10:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:10:15: TOK 4 (L = 1): "a"
../tests/invalid-chars.c:10:18: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:11:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:11:10: TOK 0 (L = 2): "c3"
../tests/invalid-chars.c:11:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:11:15: TOK 4 (L = 1): "0"
../tests/invalid-chars.c:11:18: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:12:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:12:10: TOK 0 (L = 2): "c4"
../tests/invalid-chars.c:12:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:12:15: TOK 4 (L = 2): "\0"
../tests/invalid-chars.c:12:19: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:13:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:13:10: TOK 0 (L = 2): "c5"
../tests/invalid-chars.c:13:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:13:15: TOK 4 (L = 2): "\n"
../tests/invalid-chars.c:13:19: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:14:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:14:10: TOK 0 (L = 2): "c6"
../tests/invalid-chars.c:14:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:14:15: TOK 4 (L = 2): "\r"
../tests/invalid-chars.c:14:19: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:15:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:15:10: TOK 0 (L = 2): "c7"
../tests/invalid-chars.c:15:13: TOK 1 (L = 1): "="
../tests/invalid-chars.c:15:15: TOK 4 (L = 4): "\000"
../tests/invalid-chars.c:15:21: TOK 1 (L = 1): ";"
../tests/invalid-chars.c:16:5: TOK 0 (L = 4): "char"
../tests/invalid-chars.c:16:10: TOK 0 (L = 2): "c8"
../tests/invalid-chars.c:16:13: TOK 1 (L = 1): "="
